### DISCUSSION TITLE
请问这个，新版本要自己配置 sidebar 吗？

### DISCUSSION THREAD

**Discussion by chenweigao** [vuepress-reco/vuepress-theme-reco#5] [Q&A] [answered]
https://github.com/vuepress-reco/vuepress-theme-reco-next/blob/17515f073e2ad937387751d92c021f60755f8f76/example/.vuepress/config.ts#L70

**Comment by recoluan** [comment]
> https://github.com/vuepress-reco/vuepress-theme-reco-next/blob/17515f073e2ad937387751d92c021f60755f8f76/example/.vuepress/config.ts#L70

我看你标记的代码是 navbar 里的分类，并不是 sidebar，navbar 里的分类现在确实需要手动配，后面会增加 1.x 里的自动添加。

另外 sidebar 的概念已经变化了，后面会在文档里详细说明，现在还不是稳定版本，可以先参考 example 里的配置。

**Comment by recoluan** [comment, marked as answer]
这是关于 2.0 里的侧边栏的 [文档](http://v2.vuepress-reco.recoluan.com/docs/style-default-api/series.html)。
